js/src/modals/CreateAccount/createAccount.js

```diff
@@ -200,16 +201,24 @@ export default class CreateAccount extends Component {
   }
 
   onCreate = () => {
-    const { createType } = this.state;
+    const { createType, windowsPhrase } = this.state;
     const { api } = this.context;
 
     this.setState({
       canCreate: false
     });
 
     if (createType === 'fromNew' || createType === 'fromPhrase') {
+      let phrase = this.state.phrase;
+      if (createType === 'fromPhrase' && windowsPhrase) {
+        phrase = phrase
+          .split(' ') // get the words
+          .map((w) => ['zoom', 'misjudged'].includes(w) ? w : `${w}\r`) // add \r after each (except last in dicts)
```

Comment:
So I guess we see two words here because one is coming from 1.4 dict, and the second one is in 1.5 dict.
1. We need to check if that's actually the case (that there is no new line character after last word)
2. The word from 1.4 dict *cannot* be present in 1.5 dict for this code to work.

I think it would be better to only support 1.4 (so only one word), `master` branch is known to be unstable, and we should just mention that in the release notes instead.